universe/ObjectMap.cpp

```diff
@@ -133,10 +133,11 @@ namespace {
             return false;
 
         } else {
+            // TODO Implement or remove
```

Comment:
no, this is here to generate an error if an incompatible type is used

Reply:
I misunderstood the comment due to my lack of knowledge.
I have no idea what that "using GenerateCompileError..." line does, and judging by the compiler warning, it seemed to me like a declaration of a type that was later not used. But I assume it's doing more than that but the compiler is confused.

Reply:
It doesn't do anything, really. `using X = Y;` is equiavlent to `typedef Y X;`

The point of the line is that if that constexpr else case gets compiled, it will generate a compile error.